jerry-math/jerry-math-internal.h

```diff
@@ -74,7 +76,13 @@ typedef union
 #endif /* __LITTLE_ENDIAN */
 
 #ifndef NAN
-#define NAN (0.0/0.0)
+#ifdef _MSC_VER
+#define INFINITY    ((float) (1e+300 * 1e+300)) /* 1e+300*1e+300 must overflow */
+#define NAN         ((float) (INFINITY * 0.0f))
+#else
+#define INFINITY    ((float) (1.0/0.0))
+#define NAN         ((float) (0.0/0.0))
```

Comment:
(minor) for some legacy reason, it seems that there were no spaces around the division operator. Now would be a chance to add then.